.eslintrc.json

```diff
@@ -0,0 +1,57 @@
+{
+  "ignorePatterns": ["dist", "node_modules"],
+  "rules": {
+    // "max-len": [
```

Comment:
Looks good, just remove this commented out code?

Reply:
Addressed in commit [d633e41](https://github.com/taniarascia/webpack-boilerplate/pull/40/commits/d633e41be30483029eebc44668953f39d199b711).